db/schema.rb

```diff
@@ -19,8 +22,8 @@
     t.datetime "updated_at", null: false
     t.datetime "sent_at"
     t.datetime "committed_at"
-    t.integer "event_id"
-    t.integer "guest_id"
+    t.bigint "event_id"
```

Comment:
When and why did this change happened?

Reply:
In PostgreSQL 10 and earlier, the default data type for the id column was `integer` (also known as int4).

However, starting from PostgreSQL 11, the default data type for the id column is `bigint` (also known as int8).